db/hints/manager.cc

```diff
@@ -682,6 +682,7 @@ future<> manager::end_point_hints_manager::sender::send_one_mutation(frozen_muta
 }
 
 future<> manager::end_point_hints_manager::sender::send_one_hint(lw_shared_ptr<send_one_file_ctx> ctx_ptr, fragmented_temporary_buffer buf, db::replay_position rp, gc_clock::duration secs_since_file_mod, const sstring& fname) {
+    ctx_ptr->last_sent_rp = rp;
```

Comment:
The name is confusing - it's not "last sent", it's "last attempted".

Reply:
Renamed to "last_attempted_rp"